src/features/matrix_shortcuts.ts

```diff
@@ -20,7 +20,7 @@ export const runMatrixShortcuts = (view: EditorView, ctx: Context, key: string,
 	if (!isInsideAnEnv) return false;
 
 
-	if (key === "Tab") {
+	if (key === "Tab" && !shiftKey) {
```

Comment:
This currently has no effect ("Tab" is still used to advance to the next tabstop, not "Shift+Tab") because snippet expansion/tabout is handled before matrix shortcuts in `handleKeydown`.

I'd prefer to keep the original behaviour anyway ("Tab" to move to the next tabstop), unless you have an argument against it?

Reply:
If I recall correctly I think changing `settings.matrixShortcutsEnabled && ctx.mode.blockMath` to strictlyInMath made it register the shortcut, but when I revise the code I'll confirm.

Re the second part: I think it makes more sense to have the inline commands identical to the block math commands, where Tab inserts an ampersand, since it'd also be the least cumbersome to type. 

Right now in a block math matrix,  Shift-Enter advances to the next tab stop (unless the closing $$ aren't on their own line), so we could also make Shift-Enter behave the same way inline as it does in block math. Let me know what makes the most sense here.